README.md

```diff
@@ -1,32 +1,32 @@
-# MEMEBLAST
+# BLAST
+Persistent social image sharing feed for the whole world.
 
-## Komme i gang
+## Installing
 
-virtualenv memeblast && cd memeblast && source bin/activate
-git clone git@bitbucket.org:sigveseb/memeblast.git && cd memeblast
-cp memeblast/memeblast/settings.py.example memeblast/memeblast/settings.py
-pip install -r requirements_dev.txt
-python manage.py syncdb --migrate
+virtualenv memeblast && cd blast && source bin/activate 
```

Comment:
Okay, never mind the order then – but changing only one of the instances of 'memeblast' will introduce an error.